package.json

```diff
@@ -10,40 +10,56 @@
     "node": "6.2.0"
   },
   "dependencies": {
-    "babel-preset-es2015": "6.18.0",
-    "grunt": "1.0.1",
-    "grunt-angular-templates": "1.1.0",
-    "grunt-babel": "6.0.0",
-    "grunt-cache-bust": "1.4.1",
-    "grunt-cli": "1.2.0",
-    "grunt-contrib-clean": "1.0.0",
-    "grunt-contrib-concat": "1.0.1",
-    "grunt-contrib-copy": "1.0.0",
-    "grunt-contrib-cssmin": "1.0.2",
-    "grunt-contrib-uglify": "2.0.0",
-    "grunt-usemin": "3.1.1",
-    "load-grunt-tasks": "3.5.2"
+    "angular": "1.5.8",
+    "angular-cookies": "1.5.8",
+    "angular-local-storage": "0.5.0",
+    "angular-resource": "1.5.8",
+    "angular-route": "1.5.8",
+    "angular-sanitize": "1.5.8",
+    "angular-toarrayfilter": "1.0.1",
+    "angular-ui-bootstrap": "1.3.3",
+    "angular-ui-router": "0.4.2",
+    "bootstrap": "3.3.5",
+    "font-awesome": "4.4.0",
+    "hawk": "2.3.1",
+    "jquery": "2.1.3",
+    "jquery.scrollto": "2.1.0",
+    "js-yaml": "3.7.0",
+    "lodash": "3.10.1",
+    "mousetrap": "1.4.6",
+    "ngreact": "0.3.0",
+    "react": "15.3.1",
+    "react-dom": "15.3.1",
+    "taskcluster-client": "1.6.3"
   },
   "devDependencies": {
```

Comment:
Oh actually reading back again now, I think you thought I was talking about `dependencies` whereas I was meaning `devDependencies`? eg: `copy-webpack-plugin` and others will be pulled in from the neutrino-* packages, so don't need to be defined explicitly?
https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/neutrino-preset-web/package.json#L20

Also, I've just realised - on Heroku the npm install is run with `NODE_ENV=production`, so only the `dependencies` will get installed. As such, anything required by `npm run build` will need to be in `dependencies` rather than `devDependencies`.